.github/actions/trufflehog/action.yml

```
@@ -5,8 +5,8 @@ description: "Secret Scanning"
 runs:
   using: "composite"
   steps:
-    - name: TruffleHog
-      uses: trufflesecurity/trufflehog@main
-      with:
-        head: ${{ github.head_ref || github.ref_name }}
-        extra_args: --only-verified
+  - name: TruffleHog
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)